data/members/accutivesecurity.yaml

```
@@ -28,27 +28,28 @@ press:
   language: en_US
 
 # Careers
-careers: 
-  url: 
-  feed: 
+careers:
+  url: https://www.accutive.com/careers
+  feed:
   language: en_US
 
 # Social media
 social:
-  twitter: https://twitter.com/accutivesec
-  facebook: 
-  instagram: 
-  linkedin: 
-  youtube: 
+  linkedin: https://www.linkedin.com/showcase/89817988
+  linkedin (corporate): https://www.linkedin.com/company/1113269
```

you can only have one
```suggestion
  linkedin: https://www.linkedin.com/company/1113269
```